### ISSUE TITLE
Building fails on CentOS 8

### ISSUE THREAD

**Issue by Hoeze** [hautreux/auks#52]
Some methods cannot be found:
```
../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_close'                                                                                                                      
../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_resolve_full'         
```
Probably caused by https://github.com/hautreux/auks/issues/46?

Full log:
```
Making all in auks                                                                                                                                                                           
make[3]: Entering directory '/root/rpmbuild/BUILD/auks-0.5.0_20.11.4_1/src/auks'                                                                                                            
gcc -DHAVE_CONFIG_H -I. -I../..  -I/usr/include/tirpc -I./../api   -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protecto
r-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-pr
otection -fcf-protection -DSYSCONFDIR=\"/etc/auks\" -c -o auks-auks.o `test -f 'auks.c' || echo './'`auks.c                                                                                  
/bin/sh ../../libtool  --tag=CC   --mode=link gcc  -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecor
d-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-pr
otection -DSYSCONFDIR=\"/etc/auks\"  -lkrb5 -pthread -Wl,-z,relro  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o auks auks-auks.o ../api/auks/libauksapi.la                    
libtool: link: gcc -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/r
pm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -DSYSCONFDIR=\"/etc/auk
s\" -pthread -Wl,-z -Wl,relro -Wl,-z -Wl,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o .libs/auks auks-auks.o  ../api/auks/.libs/libauksapi.so -lkrb5 -ltirpc -pthread                
../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_close'                                                                                                                      
../api/auks/.libs/libauksapi.so: undefined reference to `krb5_rc_resolve_full'                                                                                                               
collect2: error: ld returned 1 exit status                                                                                                                                                   
make[3]: *** [Makefile:397: auks] Error 1                                                                                                                                                    
make[3]: Leaving directory '/root/rpmbuild/BUILD/auks-0.5.0_20.11.4_1/src/auks'                                                                                                              
make[2]: *** [Makefile:358: all-recursive] Error 1                                                                                                                                           
make[2]: Leaving directory '/root/rpmbuild/BUILD/auks-0.5.0_20.11.4_1/src'                                                                                                                   
make[1]: *** [Makefile:413: all-recursive] Error 1                                                                                                                                           
make[1]: Leaving directory '/root/rpmbuild/BUILD/auks-0.5.0_20.11.4_1'                                                                                                                       
make: *** [Makefile:345: all] Error 2                                                                                                                                                        
error: Bad exit status from /var/tmp/rpm-tmp.nPE4wc (%build)
    Bad exit status from /var/tmp/rpm-tmp.nPE4wc (%build)
```

**Comment by pllopis** [comment]
Had the same issue. The fixes provided by #53 worked for me.

**Comment by hautreux** [comment]
fixed in 0.5.3 by b84ae7334400bf1c5dd458bdc530b2939bbd4cdc
